packages/core-tabs/core-tabs.js

```diff
@@ -43,47 +45,110 @@ export default class CoreTabs extends HTMLElement {
     }
   }
 
-  get panels () { return this.tabs.map((tab) => document.getElementById(tab.getAttribute('aria-controls'))) }
+  get panels () { return this.tabs.map(getPanelFromTab) }
 
-  get panel () { return this.panels.filter((panel) => panel && !panel.hasAttribute('hidden'))[0] }
+  get panel () { return getPanelFromTab(this.tab) }
 
+  /**
+   * @returns {[TabElement]}
+   */
   get tabs () { return queryAll('button,a', this) }
 
-  get tab () { return document.getElementById(this.panel && this.panel.getAttribute(FROM)) }
+  /**
+   * @returns {TabElement}
+   */
+  get tab () {
+    const tabAttr = this.getAttribute('tab')
+    const allTabs = this.tabs
+    const allPanels = this.panels
+
+    let tab = isInteger(parseInt(tabAttr))
```

Comment:
Blir det rett å kjøre `parseInt` inni `isInteger`? :)

Reply:
Replaced with cast to Number. Necessary to handle strings containing numbers like `'2'`